iotedgedev/template/modules/filter-module/module.json

```diff
@@ -0,0 +1,12 @@
+{

```

Comment:
Please update to use this schema: https://github.com/Azure/dotnet-template-azure-iot-edge-module/blob/master/content/dotnet-template-azure-iot-edge-module/CSharp/module.json